### ISSUE TITLE
Train indexer model

### ISSUE THREAD

**Issue by ck196** [AKSHAYUBHAT/DeepVideoAnalytics#34]
Hi,
I want to train new indexer, how can I do it.
Example fine-tune inception model (network.pb with my data)

**Comment by AKSHAYUBHAT** [comment]
I am working on integrating a triplet loss based metric learning operation which using label model will allow users to train a custom indexer.

**Comment by AKSHAYUBHAT** [comment]
Hi If you are interested, Deep Video Analytics now implements a  custom detector that you can train using region annotations & a keras YOLO model. See my recent commits, I will release a demo soon, but the UI and tasks have been implemented and it works without having to write any code. All you need is a good enough GPU and a launch a worker processing queue qtrainer.

**Comment by AKSHAYUBHAT** [comment]
Likely not in near future (next 6 months). 
Best idea is to train it outside and incorporate it via planned Custom indexer that allows arbitrary networks/input/embedding layer.
